db/migrate/20170711235422_create_articles.rb

```diff
@@ -0,0 +1,14 @@
+ROM::SQL.migration do
+  change do
+    create_table :articles do
+      primary_key :id
+      column :slug, :text, null: false, unique: true
+      column :title, :text, null: false
+      column :body, :text, null: false
+      column :published, :boolean, default: false, null: false
+      column :published_at, :timestamp
+      column :created_at, :timestamp, null: false, default: Sequel.lit("(now() at time zone 'utc')")
+      column :updated_at, :timestamp, null: false, default: Sequel.lit("(now() at time zone 'utc')")
+    end
+  end
+end
```

Comment:
Good suggestion, @choonkeat — I've just updated the README in https://github.com/dry-rb/dry-web-blog/commit/be130ae052176922ee806f036777713d133877e0